dts/arm/atmel/same70.dtsi

```diff
@@ -31,6 +32,36 @@
 	};
 
 	soc {
+		i2c0: i2c@40018000 {
+			compatible = "atmel,sam-i2c-twihs";
```

Comment:
all the i2c nodes should have:

                        #address-cells = <1>;
                        #size-cells = <0>;

Reply:
Done.